tests/coalaTest.py

```diff
@@ -322,3 +322,18 @@ def test_coala_without_color(self):
                 self.assertRegex(err, '^\[WARNING\]')
             self.assertEqual(
                 retval, 0, 'coala must return zero when there are no errors')
+
+    def test_coala_ignore_file(self):
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala',
+                    '-c', os.devnull,
+                    '--non-interactive',
+                    '-f', filename,
+                    '-i', filename,
+                    '--ignore', filename,
+                    '-b', 'LineCountTestBear')
+            self.assertIn(stdout, 'Executing section cli...\n')
```

Comment:
Are you sure this line would work? 
In [#4930 (review)](https://github.com/coala/coala/pull/4930#pullrequestreview-90337947), @userzimmermann said that the code should explicitly check that **nothing more** than 

> Executing section cli...

appears in stdout whereas I think your code would just check if that line is present in the output.

Reply:
You are right. I fixed it now.

Reply:
Great! Now we just need to figure the `LineCountTestBear` thingy.

Reply:
Also somehow there have been 5000+ additions to the code. I am not sure why.

Reply:
I think I know. I will fix it as soon as I get to my computer. On my phone atm.